### ISSUE TITLE
microk8s bin commands not available

### ISSUE THREAD

**Issue by singram** [ubuntu/microk8s#136]
Installed via
  snap install microk8s --classic
with the following result
  microk8s v1.12.0 from Canonical✓ installed

No obvious errors and kube appears to be running.

None of the microk8s binaries look to be automatically available on the path.
I've found them located under \snap\microk8s\current\ 

Aliasing also doesn't work presumably because the binaries aren't on the path to correctly.

Is this something I need to fix in the installation on my system or a bug with the snap package?

Thanks!

snap       2.35.2
snapd      2.35.2
series     16
linuxmint  18.1
kernel     4.4.0-53-generic

**Comment by ktsakalozos** [comment]
Hi @singram ,

All snap binaries should be available under /snap/bin/ and microk8s should not be an exception. Can you please make sure you have /snap/bin in your PATH?

Thank you for using microk8s.

**Comment by singram** [comment]
Thanks for the assistance, appreciated.
PATH is now fixed and the issue is resolved.

**Comment by cyberpau** [comment]
> Thanks for the assistance, appreciated.
> PATH is now fixed and the issue is resolved.

Do you mind if you can share how did you fixed the issue? I'm having some problems installing minikube on my vmware so I'm exploring microk8s. Thanks in advance!

**Comment by reginapizza** [comment]
Came here looking for a solution to the same problem and couldn't find it, so wanted to include my solution for others to try in case they are in the same situation: 

In the install steps, I did not set my PATH for snapd. 

To add /snap/bin to your PATH, run `export PATH="$PATH:/snap/bin"`,  and check that it's in your PATH with `echo $PATH`

For reference, I got my [solution from this site](https://discuss.getsol.us/d/894-snap-bin-not-included-in-your-path).

Hope this helps somebody!
